.nvmrc

```diff
@@ -0,0 +1 @@
+node -v
```

Comment:
That's funny that this actually prompted Cloudflare to download the latest version of node and it worked without an issue.